exercise3.js

```diff
@@ -86,50 +114,106 @@
 const numbers = [2,23,1,2,1,1,1,2,2.5,20,200,2000,,{k:1},20000,19999,1878,140,23,4,"sk",true,true,"true-dat","nice","one","two","three","3","tea",[]];
 
 function maxNumber(numbers) {
-    //your code...
-}
+  //your code...
+  const truthys = numbers.filter(el => el);
+  const primitives = truthys.filter(n => typeof n != 'object');
+  const noBools = primitives.filter(p => typeof p != 'boolean');
+  const numStrings = ["zero","one","two","three"];
+  const numWithStrings = noBools.map(nb => numStrings.includes(nb) ? numStrings.indexOf(nb) : nb);
+  const numWithStrings2 = numWithStrings;
+  numWithStrings2[24] = Number(numWithStrings[24]);
```

Comment:
Awesome work @AbigailSwigert , adding the case for the string '3'.  Really impressed!  😀 

the only issue is, if you are trying to create a new array with `numWithStrings2`, how you did it will **not** work because you are pointing to the same reference in memory.  you should instead spread in the values of numWithStrings.

`const numWithStrings2 = [. . .numWishStrings]`